ui/app/common/components/pnc-admin-panel/pnc-admin-panel.html

```diff
@@ -60,7 +82,8 @@ <h4 class="modal-title" id="adminPanelLabel">Admin Panel</h4>
   <div class="modal-dialog">
     <div class="modal-content">
       <div class="modal-header">
-        <button type="button" class="close" data-dismiss="modal" aria-hidden="true" ng-click="$ctrl.clearMaintenanceValidation()">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true"
```

Comment:
format edits like this should be avoided in the future if there is no specific reason to do it

Reply:
Sure, will keep that in mind